floodviz/static/js/hydrograph.js

```diff
@@ -0,0 +1,55 @@
+document.addEventListener("DOMContentLoaded", function(event) { 
```

Comment:
If we start doing any serious DOM selection/manipulation or adding event handlers, I think we should start using jquery, but I'm fine with leaving this as is for now.